app/controllers/application_controller.rb

```diff
@@ -13,18 +13,22 @@ class ApplicationController < ActionController::Base
   # Custom Error Methos
   module SnapException
     class AccessDenied < StandardError; end
-  end 
+  end
 
   # Clean Error Handler Methods
   # Use these in place of handling errors individually.
-  def access_denied # Equivalent 401
-    raise SnapException::AccessDenied.new('You don\'t have permission to view this.')
+  def access_denied(msg=nil)  # Equivalent 401
```

Comment:
Unnecessary spacing detected.<br>Surrounding space missing in default value assignment.